exporter/awsxrayexporter/internal/translator/aws.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"go.opentelemetry.io/collector/pdata/pcommon"
 	conventions "go.opentelemetry.io/collector/semconv/v1.12.0"
+	conventionsv127 "go.opentelemetry.io/collector/semconv/v1.27.0"
```

Comment:
Perhaps name the latest version `conventions` and rename v1.12.0 to `conventionsv112`? Would it make sense?
```suggestion
	conventionsv112 "go.opentelemetry.io/collector/semconv/v1.12.0"
	conventions "go.opentelemetry.io/collector/semconv/v1.27.0"
```

Reply:
Updated the PR with renaming.